.github/actions/fetch_bladebit_harvester.sh

```diff
@@ -89,5 +91,9 @@ if [[ "${artifact_ext}" == "zip" ]]; then
 else
   pushd "${dst_dir}"
   tar -xzvf "../../${artifact_name}"
+  if [[ "${host_os}" == "linux" ]] && [[ "${host_arch}" == "x86-64" ]]; then
+    # On linux clear the GNU_STACK executable bit for glibc 2.41 compat
+    execstack -c lib/libbladebit_harvester.so
+  fi
```

Comment:
i think there ought to be a note explaining why this is needed and at what point it should be removed.  should have a `# TODO:`.